Shape Functions/index.js

```diff
@@ -0,0 +1,27 @@
+function getAreaOfCircle(radius) {
+    return Math.PI * radius * radius;
+}
+
+let circleArea = getAreaOfCircle(5);
+console.log("Area of Circle:", circleArea);
+
+
+function getCircumferenceofCircle(radius) {
```

Comment:
Function declarations are okay to start out with but eventually we will prefer to use function expressions:

```
// The basic form looks like this

const getCircleCircumference = (radius) => {
  return 2 * Math.PI * radius
}

// And there are some convenient nuances that make writing them easier:
// when only one paramter, parenthesis are optional
// when only one line of function body, everything can go on one line and the `return` keyword is optional

const getCircleCircumference = radius => 2 * Math.PI * radius
```